src/adapters/fsFileSystem.ts

```diff
@@ -12,14 +16,14 @@ export const fsFileSystem: FileSystem = {
     },
     readFile: async (filePath: string) => {
         try {
-            return (await fs.promises.readFile(filePath)).toString();
+            return (await readFile(filePath)).toString();
         } catch (error) {
             return error;
         }
     },
     writeFile: async (filePath: string, contents: string) => {
         try {
-            return fs.promises.writeFile(filePath, contents);
+            return writeFile(filePath, contents);
```

Comment:
@JoshuaKGoldberg let me know if this looks right! (write ;D)

Reply:
@egiurleo perfect 👌🕺💃